### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by asiekierka** [Samuel-Harbord/SpectriteMod#17]
http://paste.asie.pl/5QhP

**Comment by Flashfyre** [comment]
Aaaaand that's from using old mappings... I was hoping it'd just be the dev environment but obviously not. Thanks for informing me before I went to work.

**Comment by Flashfyre** [comment]
Should be fixed in 1.4.6.

Edit: Looked at it again and that wouldn't have fixed it. Fixed for real in 1.4.7. If it still doesn't work it's because I was rushing so I could catch my bus. xD

Edit #2: Seems like it still crashes. Looks like a bunch of obfuscated method/field names changed for some reason. I'll have to fix it tonight.

Edit #3: Yep, I was definitely in a rush, I changed the mapping to the one for a field of the same name instead of the method. It might just be this one then.

Edit #4: Ok, the original problem was that I used the obfuscated name for "renderShadowSingle" instead of "renderShadow". Shouldn't be a problem fixing this then.
